__tests__/units/fee-calculator.test.ts

```diff
@@ -0,0 +1,385 @@
+import { FileManager } from "../../src/file-manager";
+import { FeeCalculator } from "../../src/fee-calculator";
+import {
+  DistributorType,
+  DistributorsData,
+  BalanceData,
+  FeeReport,
+} from "../../src/types";
+
+jest.mock("../../src/file-manager");
```

Comment:
Lets not mock the filemanager - lets actually write data to it during the test. We should create and clean up a test data folder at the start/end of the script